pkg/util/bitflag_test.go

```diff
@@ -25,12 +27,8 @@ const (
 	FlagD
 )
 
-type bitFlagSuite struct{}
-
-var _ = check.Suite(&bitFlagSuite{})
-
-func (b *bitFlagSuite) TestExample(c *check.C) {
-	defer testleak.AfterTest(c)()
+func TestExample(t *testing.T) {
+	defer testleak.NewAfterTest(t)()
```

Comment:
i think we do not need `testleak` after migration to testify, try https://github.com/pingcap/tiflow/blob/master/pkg/leakutil/leak_helper.go